lib/utilities/serverAccessLogger.js

```diff
@@ -0,0 +1,332 @@
+const { Werelogs } = require('werelogs');
```

Comment:
To be discussed but in general I prefer the logic that If dir OR file missing we disable server access logging.

It has a clear, explicit behavior: if the admin wants access logs, they must: create the directory, create the file, set serverAccessLogs.enabled = true

This avoids implicit side effects like silently creating a directory, silently creating a file, silently logging somewhere unexpected.
I prefer our admin/deployer to have full control.